src/app/app.component.spec.ts

```diff
@@ -15,17 +15,4 @@ describe('AppComponent', () => {
     const app = fixture.componentInstance;
     expect(app).toBeTruthy();
   });
-
```

Comment:
if you moved the other app files, this one probably needs to move as well.

Reply:
this tests the app.component itself, does not need to be moved